community-containers/memories/memories.json

```diff
@@ -0,0 +1,33 @@
+{
+    "aio_services_v1": [
+        {
+            "container_name": "nextcloud-aio-memories",
+            "display_name": "Transcoding container for memories",
+            "documentation": "https://github.com/nextcloud/all-in-one/tree/main/community-containers/memories",
+            "image": "radialapps/go-vod",
+            "image_tag": "latest",
+            "internal_port": "47788",
+            "restart": "unless-stopped",
+            "environment": [
+                "TZ=%TIMEZONE%",
+                "NEXTCLOUD_HOST=https://%NC_DOMAIN%"
+            ],
+            "volumes": [
+                {
+                    "source": "%NEXTCLOUD_DATADIR%",
+                    "destination": "/mnt/ncdata",
+                    "writeable": false
+                }
+            ],
+            "devices": [
+                "/dev/dri"
+            ],
+            "nextcloud_exec_commands": [
+                "php /var/www/html/occ app:install memories",
+                "php /var/www/html/occ app:enable memories",
```

Comment:
These don't seem to execute for me, so I needed to do the config manually. This could be something on my side though because I've a weird setup. Also, I'd prefer not auto-installing memories here, but leave that to the admin to do.

Reply:
Yes, this is likely due to your test setup. The commands above only get executed if the nextcloud container can reach `https://$NC_DOMAIN:443` which is not the case in your testing setup.

I would actually like to keep it. Is that fine for you?

Reply:
Ah okay.

> I would actually like to keep it. Is that fine for you?

Sure I don't mind. Maybe worth a line in the description then that it also installs the app? I'm also assuming this is a no-op if already installed.

Reply:
> Sure I don't mind. Maybe worth a line in the description then that it also installs the app?

I may add this as a follow-up 👍

> I'm also assuming this is a no-op if already installed.

Yes